lucene/codecs/src/java/org/apache/lucene/codecs/uniformsplit/UniformSplitTermsReader.java

```diff
@@ -68,16 +68,18 @@
    *                     It can be used for decompression or decryption.
    */
   public UniformSplitTermsReader(PostingsReaderBase postingsReader, SegmentReadState state, BlockDecoder blockDecoder) throws IOException {
-    this(postingsReader, state, blockDecoder, NAME, VERSION_START, VERSION_CURRENT,
+    this(postingsReader, state, blockDecoder, new FieldMetadata.Serializer(), NAME, VERSION_START, VERSION_CURRENT,
```

Comment:
Since it appears the default instance has no state, can't we re-use a shared static instance of this one?

Reply:
Good point.